src/actions/replyMentions.ts

```diff
@@ -1,20 +1,21 @@
 import { T } from '../helpers/twitterConnect'
 import { postTweet } from '../helpers/twitterActions'
 import { getRandomJoke } from '../helpers/utils'
+import { Twitter } from 'twit'
 
 function trackMentions (twitterHandler): void {
   const stream = T.stream('statuses/filter', { track: twitterHandler })
 
-  stream.on('tweet', (tweet) => {
-    const asker: string = `@${tweet.user.screen_name}`
+  stream.on('tweet', (tweet: Twitter.Status): void => {
+    const requester: string = `@${tweet.user.screen_name}`
```

Comment:
thank you for this. I was not too happy with asker. wrote that like 5 years back, but could not think of something better.